### PR TITLE
add registerElement to NativeScriptVueConstructor

### PR DESCRIPTION
**Pull request by gimyboya**
Fix:  `property 'registerElement' does not exist on type 'NativeScriptVueConstructor'.`

Currently, the function type is defined but was not included in the constructor.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -1,16 +1,12 @@
 // import vue.js typings
 // import Vue from 'vue';
 import { Vue, VueConstructor } from 'vue/types/vue'
-import { Page, NavigationEntry } from 'tns-core-modules/ui/frame/frame'
+import { Page, NavigationEntry, Size } from 'tns-core-modules/ui/frame/frame'
 import { View } from 'tns-core-modules/ui/core/view'
 
-export interface NavigationEntryVue extends NavigationEntry {
```
Comment (jjstrat3) [inline index.d.ts]: Your branch looks to be out of date, as you've removed changes to add the enhanced type `NavigationEntryVue` which was added to address this issue: #409

Comment (gimyboya) [reply index.d.ts]: I have updated the pr, please review @jjstrat3

**Comment by jjstrat3** [comment]
I guess only @rigor789 has merge privileges?

**Comment by gimyboya** [comment]
@rigor789 any heads up on this?

**Pull request merged**